libr/asm/arch/mips/gnu/mips-dis.c

```diff
@@ -620,6 +620,16 @@ parse_mips_dis_option (const char *option, unsigned int len)
   val = option + (optionlen + 1);
   vallen = len - (optionlen + 1);
 
+  if (strncmp ("abi", option, optionlen) == 0
+	&& strlen ("abi") == optionlen) {
+	chosen_abi = choose_abi_by_name (val, vallen);
+	if (chosen_abi != NULL) {
```

Comment:
!=NULL is unnecessary

Reply:
If the user executes "e asm.features=XXX", the choose_abi_by_name function may be return NULL

Reply:
@junchao-loongson what he means is that we usually do:
```suggestion
	if (chosen_abi) {
```